cdap-proto/src/main/java/co/cask/cdap/proto/profile/ProfileInfo.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.proto.profile;
+
+import java.util.Objects;
+
+/**
+ * Information about a profile.
+ */
+public class ProfileInfo {
```

Comment:
I think this would only ever be used to create a profile, so we can name it ProfileCreateRequest.